diffkemp/simpll/Utils.h

```diff
@@ -129,4 +129,12 @@ std::string getIdentifierForValue(const Value *Val,
 /// methods.
 std::string valueToString(const Value *Val);
 
+/// Get a string matching the current indentation level.
+/// \param prefixChar Indentation prefix character, defaults to space.
+std::string getIndent(const char prefixChar = ' ');
+
+/// Change the indentation level based on offset.
+/// \param offset Indentation level offset.
+void offsetIndentLevel(const int offset);
```

Comment:
Since this is always used with 1 or -1, I'd suggest replacing it with two functions:
`increaseDebugIndent` and `decreaseDebugIndent`.